pep/translations/en-us.json

```diff
@@ -72,7 +72,19 @@
             "addToFavorites": "Add to favorites",
             "removeFromFavorites": "Remove from favorites",
             "addToReadLater": "Add to read later",
-            "removeFromReadLater": "Remove from read later"
+            "removeFromReadLater": "Remove from read later",
+            "notifications": {
+                "success": {
+                    "addToFavorites": "Document successfully added to favorites",
+                    "removeFromFavorites": "Document successfully removed from favorites",
+                    "addToReadLater": "Document successfully added to read later list",
+                    "removeFromReadLater": "Document successfully removed from read later list"
```

Comment:
I think that using "successfully" in success messages is usually overly verbose and not needed. Its a bit cleaner w/o:

```
"Document added to favorites"
"Document removed from favorites"
//etc...
```